### DISCUSSION TITLE
Add empty space to beginning and end of Declaration and Implementation section

### DISCUSSION THREAD

**Discussion by camais** [Roald87/TcBlack#59] [Ideas]
This way, reading the lines are a bit easier when doing a diff in the TcPOU xml view

        <Declaration><![CDATA[
PROGRAM MAIN
VAR
   testBit : BOOL;
END_VAR
]]></Declaration>

vs

    <Declaration><![CDATA[PROGRAM MAIN
VAR
   testBit : BOOL;
END_VAR]]></Declaration>

**Comment by Roald87** [comment]
Hi @camais, thanks for your input!

The extra white line for the end of the declaration part is already implemented. For the implementation part I will also do this. See also the [style guide](https://github.com/Roald87/TcBlack/blob/master/docs/style.md#empty-lines).

As for the extra white line at the start I'm not sure yet. It was also shortly discussed in this open PR https://github.com/Roald87/TcBlack/pull/41#discussion_r460530784.
